src/components/form/form.pw.tsx

```diff
@@ -0,0 +1,387 @@
+      await expect(warnings).toHaveCSS("color", "rgb(167, 72, 0)");
+      await expect(warnings).toHaveText(`${warningCount} warnings`);
+      await expect(warnings).toBeVisible();
+    });
+  });
+
+  ([
+    ["30px", 30],
+    ["100px", 100],
+    ["200px", 200],
+  ] as [FormProps["height"], number][]).forEach(([height, heightVal]) => {
+    test(`should render with height of ${height}`, async ({ mount, page }) => {
+      await mount(<FormComponent height={height} />);
+
+      const form = getComponent(page, "form");
+      await assertCssValueIsApproximately(form, "height", heightVal);
+    });
+  });
+
+  ([
+    [false, 98],
+    [true, 1366],
+  ] as [FormProps["fullWidthButtons"], number][]).forEach(
+    ([bool, buttonWidth]) => {
+      test(`should render with buttonWidth of ${buttonWidth} when fullWidthButtons prop is ${bool}`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<FormWithFullWidthButtons fullWidthButtons={bool} />);
+
+        const submitButton = getComponent(page, "button").filter({
+          hasText: "submit",
+        });
+        await assertCssValueIsApproximately(submitButton, "width", buttonWidth);
+      });
+    }
+  );
+});
+
+test.describe("Accessibility tests for Form component", () => {
+  test(`should pass tests for DefaultWithStickyFooter example`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<DefaultWithStickyFooter />);
+
+    await checkAccessibility(page);
+  });
+
+  test(`should pass tests for FormAlignmentExample`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<FormAlignmentExample />);
+
+    await checkAccessibility(page);
+  });
+
+  test(`should pass tests for InDialog example`, async ({ mount, page }) => {
+    await mount(<InDialog />);
+
+    const dialogButton = dataComponentButtonByText(page, "Open Preview");
+    await dialogButton.click();
+    await checkAccessibility(page);
+  });
+
+  test(`should pass tests for InDialogFullScreen example`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<InDialogFullScreen />);
+
+    const dialogButton = dataComponentButtonByText(page, "Open Preview");
+    await dialogButton.click();
+    await checkAccessibility(page);
+  });
+
+  test(`should pass tests for InDialogFullScreenWithStickyFooter example`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<InDialogFullScreenWithStickyFooter />);
+
+    const dialogButton = dataComponentButtonByText(page, "Open Preview");
+    await dialogButton.click();
+    await checkAccessibility(page);
+  });
+
+  test(`should pass tests for InDialogWithStickyFooter example`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<InDialogWithStickyFooter />);
+
+    const dialogButton = dataComponentButtonByText(page, "Open Preview");
+    await dialogButton.click();
+    await checkAccessibility(page);
+  });
+
+  test(`should pass tests for OverrideFieldSpacing example`, async ({
```

Comment:
I don't think this needs an accessibility test. Having spacing overridden would not cause a violation.